<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransformInfoForm</class>
 <widget class="QWidget" name="TransformInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QLineEdit" name="binResolutionEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="binResolutionLabel">
       <property name="text">
        <string>Hz/bin</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="maxHzEdit"/>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="minHzEdit"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="minHzLabel">
       <property name="text">
        <string>Min hz</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="maxHzLabel">
       <property name="text">
        <string>Max hz</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
